pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -202,19 +202,11 @@ func (db *DB) GetBandwidthAllocationBySignature(signature []byte) ([][]byte, err
 }
 
 // AddTTLToDB adds TTL into database by id
-func (db *DB) AddTTLToDB(id string, expiration int64) error {
+func (db *DB) AddTTLToDB(id string, expiration, size int64) error {
```

Comment:
Rename this to AddTTL, the `DB` is already implicit due to receiver.

Reply:
👍